### PR TITLE
Ansi colors

### PR DESCRIPTION
**Pull request by NathanBaulch**
Colors! This branch also includes my changes in #34 since colors are particularly useful for differentiating between different series in the same plot.
AnsiRainbow

### CODE REVIEW

**Comment by guptarohit** [comment]
Thanks, @NathanBaulch for your time to contribute to this change, it was a long time due. :tada: :cake:

I was thinking if we could also add flags for this color support in [CLI utility](https://github.com/guptarohit/asciigraph/blob/master/cmd/asciigraph/main.go) it would be really great addon!

**Comment by NathanBaulch** [comment]
Good idea, I went with `-l` since all the other letters in "color" are taken.

**Comment by guptarohit** [comment]
Hey @NathanBaulch, I noticed while plotting the first value i.e. ┼ , it's also changing the color of y-axis for that value, is this how you expected it to be? I'd like this to be AxisColor.
image

**Comment by NathanBaulch** [comment]
> is this how you expected it to be?

It was intentional yes, but I'm not too fussed either way. I wonder what `kroitor` does?

**Comment by guptarohit** [comment]
> It was intentional yes, but I'm not too fussed either way. I wonder what `kroitor` does?

I just checked, looks like he's also making color after y-axis
Screenshot from 2022-05-03 17-09-50

**Review by guptarohit — APPROVED** [review]

**Comment by guptarohit** [comment]
Hey @NathanBaulch, could you please review the changes once, then I'll merge these changes

File: cmd/asciigraph/main.go [hunk 1]
```diff
@@ -39,6 +44,42 @@ func main() {
...
+			return errors.New("unrecognized seriesColor")
+		} else {
+			seriesColor = c
+			return nil
+		}
+	})
+
+	flag.Func("cc", "`caption color` of the plot", func(str string) error {
+		if c, ok := asciigraph.ColorNames[str]; !ok {
+			return errors.New("unrecognized seriesColor")
```
Comment (NathanBaulch) [inline cmd/asciigraph/main.go:58]: should this be "unrecognized caption color"?

**Comment by NathanBaulch** [comment]
Looks good @guptarohit, just my comment above about the new cmd color flag errors all being the same.

File: cmd/asciigraph/main.go (hunk 1 above)
Comment (guptarohit) [reply cmd/asciigraph/main.go]: good catch! :+1: Fixed.

**Comment by NathanBaulch** [comment]
FWIW, the color names I went with are the standard SVG/CSS ones, as listed here: https://www.w3.org/TR/SVG11/types.html#ColorKeywords
In case you wanted a more "official" list rather than pointing people to `color.go` here.

**Comment by guptarohit** [comment]
cool, it make sense to use this link instead :+1: :tada: Thanks :)

**Pull request merged**